### ISSUE TITLE
Databond support

### ISSUE THREAD

**Issue by ashish10677** [razor-network/razor-go#685]
# Rationale

Incase a block is not confirmed in the previous epoch, razor go would need to do offchain calculations on occurrence for all data bonds and check whether they need to be reported upon in this epoch.

# Implementation

**Comment by ashish10677** [comment]
More clarity needed on this @SamAg19

**Comment by SkandaBhat** [comment]
Was this clarified? @SamAg19 @ashish10677

**Comment by ashish10677** [comment]
Yes @SkandaBhat. We had a call yesterday and this was sorted out.

**Comment by ashish10677** [comment]
@Yashk767 can you please add more to this issue so that it'll be easier for you in implementation.
